microdnf/context.cpp

```diff
@@ -76,9 +76,47 @@ bool userconfirm(libdnf::ConfigMain & config) {
 
 namespace {
 
-class MicrodnfRepoCB : public libdnf::repo::RepoCallbacks {
+// The "quiete" repo callback class implements callback only for importing repository key, no progress information.
+class MicrodnfQuietRepoCB : public libdnf::repo::RepoCallbacks {
```

Comment:
Could you then call this callback class e.g. `KeyImportCallbacks` or `MicrodnfKeyImportRepoCB` if you want to keep close to the original naming convention? That is to name it according to what it does instead of it's purpose to have a quiet mode.

And then the inheriting class could be called e.g. `ProgressAndImportCallbacks`... but I'm also fine with that one's current name if you prefer that.

Also, typo "quiete" in the comment above.

Reply:
The names are microdnf internal. No API. But I renamed them.